templates/_schemas/details/default.twig

```diff
@@ -2,14 +2,28 @@
 
 
 
+{% set graph = create('Spatie\\SchemaOrg\\Graph') %}
+{% set seomateMetaData = craft.seomate.getMeta({
+  config: { includeSitenameInTitle: false },
+}|merge(seomate ?? {}, true)) %}
 {% set entryImages = entryImages ?? craft.assets.relatedTo(entry).kind('image').all() %}
 
 {% set defaultSchemaProps = {
-  title: entry.title,
+  title: seomateMetaData.meta.title ?? entry.title,
   url: entry.url,
-  dateCreated: entry.postDate,
-  dateUpdated: entry.dateUpdated,
+  mainEntityOfPage: mainEntityOfPage ?? null,
+  dateUpdated: entry.dateUpdated ?? null,
+  datePublished: entry.postDate ?? null,
+  dateCreated: entry.postDate ?? null,
   images: entryImages|filter|values,
+  authors: [{
+    title: siteInformation.seoTitle ?? siteName,
```

Comment:
What is the `siteInformation` variable and where is it coming from?

Reply:
`siteInformation` is a global field. Removed it as that is also an assumption, chances are the user won't have it in the project.